its/src/test/kotlin/org/sonarlint/intellij/its/tests/IdeaTests.kt

```diff
@@ -69,11 +70,13 @@ import org.sonarlint.intellij.its.utils.ProjectBindingUtils.Companion.bindProjec
 import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clearConnections
 import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clearConnectionsAndAddSonarQubeConnection
 import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.clickPowerSaveMode
+import org.sonarlint.intellij.its.utils.SettingsUtils.Companion.toggleRule
 import org.sonarqube.ws.client.WsClient
 import org.sonarqube.ws.client.issues.DoTransitionRequest
 import org.sonarqube.ws.client.issues.SearchRequest
 import org.sonarqube.ws.client.settings.SetRequest
 
+//In order to run these test change the url triggerOpenHotspotRequest to 64121
```

Comment:
```suggestion
// In order to run these test change the url triggerOpenHotspotRequest to 64121
```
Technically it can be another port than 64121, it depends on how many instances of IntelliJ you have opened, maybe you can adapt the comment?